jupyter_client/tests/test_kernelapp.py

```diff
@@ -47,18 +47,10 @@ def test_kernelapp_lifecycle():
             raise AssertionError("No started file created in {} seconds"
                                  .format(WAIT_TIME))
 
-        # Connection file should be there by now
-        files = os.listdir(runtime_dir)
-        assert len(files) == 1
-        cf = files[0]
-        assert cf.startswith('kernel')
-        assert cf.endswith('.json')
-
```

Comment:
This test should still pass. Can you confirm that the default connection filename is still unchanged?

Reply:
OK added the check again. Should be fine now.